File: AndroidApps/PSharedPreference/app/src/main/java/com/jinco/psharedpreference/MainActivity.java
package com.jinco.psharedpreference;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    EditText name, email;
    Button btn, btn2;
    SharedPreferences sp;
    String nameInput, emailInput;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        name = findViewById(R.id.editTextTextPersonName);
        email = findViewById(R.id.editTextTextPersonName2);
        btn = findViewById(R.id.button);
        btn2 = findViewById(R.id.button2);
        sp = getSharedPreferences("p", Context.MODE_PRIVATE);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                nameInput = name.getText().toString();
                emailInput = email.getText().toString();

                SharedPreferences.Editor editor = sp.edit();
                editor.putString("name", nameInput);
                editor.putString("email", emailInput);
                editor.commit();
                Toast.makeText(MainActivity.this, "Information Saved.", Toast.LENGTH_LONG).show();
            }
        });

        btn2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, OtherPage.class);
                startActivity(intent);
            }
        });
    }
}

File: AndroidApps/FormPractice/app/src/main/java/com/jinco/formpractice/MainActivity.kt
package com.jinco.formpractice

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.*

class MainActivity : AppCompatActivity() {
    var name:EditText? = null
    var contact:EditText? = null
    var email:EditText? = null
    var immediateStart:CheckBox? = null
    var jobTitle:Spinner? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        name = findViewById(R.id.et_contact_name)
        contact = findViewById(R.id.et_contact_number)
        email = findViewById(R.id.et_contact_email)
        immediateStart = findViewById(R.id.cb_immediate_start)
        jobTitle = findViewById(R.id.sp_job_title)
        var btnSubmit:Button = findViewById(R.id.btn_submit)

        btnSubmit.setOnClickListener {
            Toast.makeText(this, "hi", Toast.LENGTH_LONG).show()
            Log.e("test", "hi")
        }

    }
}